visualization/dashboard/src/Localization/en.json

```diff
@@ -423,5 +432,15 @@
         "_TreatAsCategorical.comment": "a checkbox label to treat integers as categories instead of as numbers",
         "numOfBins":"Number of bins",
         "_numberOfBins.comment": "the number of groups (bins) to place all values into"
+    },
+    "FilterOperations": {
```

Comment:
Interesting, I don't know if these need translation (other than "includes").

Reply:
adding translation for "includes" but how do you think I should handle others?